tests/jax/components/training/losses_test.py

```diff
@@ -185,13 +185,14 @@ def test_mapg_loss(
         agent: jnp.array([3.0, 3.0, 3.0, 3.0])
         for agent in {"agent_0", "agent_1", "agent_2"}
     }
-
+    policy_states = {"agent_0": None, "agent_1": None, "agent_2": None}
```

Comment:
for completion, we should probably test with policy_states as None so that we are still testing feedforward

Reply:
In the code we pass individual `policy_states` per agent. So this is why I use a dictionary over `None` values. Do you think this is fine? Or should do we something else?
```
policy_grads[agent_key], loss_info_policy[agent_key] = jax.grad(
                policy_loss_fn, has_aux=True
            )(
                policy_params[agent_net_key],
                 policy_states[agent_key],
                observations[agent_key].observation,
                actions[agent_key],
                behaviour_log_probs[agent_key],
                advantages[agent_key],
            )

```

Reply:
in `losses.py` we have a check `if policy_states: #recurrent actor else: #feedforward actor code` so this test would never go into the `else` and test the feedforward code

Reply:
That is inside the `policy_loss_fn` function. So that `policy_states` is the agent specific policy_state which is `None` in this case.

Reply:
so it will only run the feedforward code then?

Reply:
same for `step_test.py`

Reply:
Thanks